.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 npm-debug.log
+.idea
```

Comment:
Let's remove this. We should avoid adding personal development environment related things.